icon_validator/rules/workflow_validators/workflow_description_validator.py

```diff
@@ -0,0 +1,27 @@
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class WorkflowDescriptionValidator(KomandPluginValidator):
+
+    @staticmethod
+    def validate_workflow_description_exists(spec):
+        if "description" not in spec.spec_dictionary():
+            raise ValidationException("Plugin description is missing.")
```

Comment:
Is that wrong? Is it supposed to be "Workflow description is missing?"

Reply:
See below as well.

Reply:
lol good old copy past